### PR TITLE
Created GetDictionaryKeys, updated readme, updated test cases and pro…

### PR DESCRIPTION
**Pull request by JLongUiPathTAM**
Created GetDictionaryKeys, updated readme, updated test cases and project file for legacy and windows for get dictionary keys. ~Joe L.

### CODE REVIEW

File: StudioTestProjects/UiPath.Activities.Extensions.TestProjectLegacy/TestGetDictionaryKeys.xaml
Comment (k2zinger) [inline StudioTestProjects/UiPath.Activities.Extensions.TestProjectLegacy/TestGetDictionaryKeys.xaml:1]: Change this test to be in progress

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx [hunk 1]
```diff
@@ -465,4 +465,72 @@
...
+    <comment>error message; {0} is the name of a missing property</comment>
+  </data>
+  <data name="ValidationExclusiveProperties_Error" xml:space="preserve">
+    <value>Please provide a value either for {0} or {1}, but not both.</value>
+    <comment>error message; {0} &amp; {1} are mutually exclusive activities</comment>
+  </data>
+  <data name="ValidationOverloadGroup_Error" xml:space="preserve">
+    <value>Please provide values for both {0} and {1}.</value>
+    <comment>error message; {0} &amp; {1} are complementary properties</comment>
+  </data>
```
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:503]: remove duplicative keys from the wizard

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx [hunk 2]
```diff
@@ -465,4 +465,72 @@
...
+    <comment>property name</comment>
+  </data>
+  <data name="GetDictionaryKeys_In_Dictionary_Description" xml:space="preserve">
+    <value>The dictionary you want to retrieve the keys from.</value>
+    <comment>property description</comment>
+  </data>
+  <data name="Input_Category" xml:space="preserve">
+    <value>Input</value>
+    <comment>category</comment>
+  </data>
```
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:523]: remove duplicative keys from the wizard

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx [hunk 3]
```diff
@@ -465,4 +465,72 @@
...
+    <comment>property name</comment>
+  </data>
+  <data name="GetDictionaryKeys_Out_Keys_Description" xml:space="preserve">
+    <value>The list of keys from the provided dictionary.</value>
+    <comment>property description</comment>
+  </data>
+  <data name="Output_Category" xml:space="preserve">
+    <value>Output</value>
+    <comment>category</comment>
+  </data>
```
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:535]: remove duplicative keys from the wizard

**Review by k2zinger — CHANGES_REQUESTED** [review]
changes requested

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx (hunk 2 above)
Comment (JLongUiPathTAM) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:523]: Removed 520-523 Input_Category duplicate key

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx (hunk 1 above)
Comment (JLongUiPathTAM) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:503]: Removed 463-503, multiple duplicate keys

**Pull request merged**